projects/element-ng/common/models/status-type.model.ts

```diff
@@ -16,6 +16,9 @@ export interface StatusIcon {
   severity: number; // for sorting
 }
 
+/**
+ * @deprecated Use the injection token `STATUS_ICON_CONFIG` instead.
```

Comment:
While this is deprecated, the other is experimental. I think one annotation should be removed.

Reply:
I removed the deprecation, as this is more consitant since the other one using experimental stuff.